input/templates/partials/myaccount/my-address-book-adress.hbs

```diff
@@ -0,0 +1,18 @@
+<div>{{address.title}} {{address.firstName}} {{address.lastName}}</div>
+<div>{{address.streetName}}</div>
+<div>{{address.additionalStreetInfo}}</div>
+<div>{{address.region}}</div>
+<div>{{address.postalCode}}</div>
+<div>{{address.country}}</div>
+<div class="address-book-edit-delete">
+  <a href="{{address.editUrl}}">
+    <img src="{{meta.assetsPath}}/img/edit-1.png" alt="edit icon">
+    {{i18n "my-account:buttons.edit"}}
+  </a>
+</div>
+<div>
+  <a href="{{address.deleteUrl}}">
```

Comment:
Here it was missing the delete URL, which follows the same principle as the edit URL :)